README.md

```diff
@@ -51,17 +54,17 @@ with open('typesystem.xml', 'rb') as f:
     typesystem = load_typesystem(f)
     
 with open('cas.xml', 'rb') as f:
-   cas = load_cas_from_xmi(f, typesystem=typesystem)
+    cas = load_cas_from_xmi(f, typesystem=typesystem)
    
 Token = typesystem.get_type('cassis.Token')
 
 tokens = [
-    Token(xmiID=13, sofa=1, begin=0, end=3, id='0', pos='NNP'),
-    Token(xmiID=19, sofa=1, begin=4, end=10, id='1', pos='VBD'),
-    Token(xmiID=25, sofa=1, begin=11, end=14, id='2', pos='IN'),
-    Token(xmiID=31, sofa=1, begin=15, end=18, id='3', pos='DT'),
-    Token(xmiID=37, sofa=1, begin=19, end=24, id='4', pos='NN'),
-    Token(xmiID=43, sofa=1, begin=25, end=26, id='5', pos='.'),
+    Token(begin=0, end=3, id='0', pos='NNP'),
```

Comment:
I created a simple type system to test stuff with, it is in https://github.com/dkpro/dkpro-cassis/blob/master/tests/test_files/typesystems/small_typesystem.xml .

Reply:
Do you plan on generally materializing types from the type system as Python classes in a JCas way?

Reply:
It would be great if you could include a test and maybe an example which uses the DKPro Type system (or at least a part of it, e.g. the [segmentation types](https://github.com/dkpro/dkpro-core/blob/master/dkpro-core-api-segmentation-asl/src/main/resources/desc/type/LexicalUnits.xml)). I could also try to generate an XML which contains the *entire* DKPro Core type system for the development of Cassis. Normally we do not have such an overall XML because the type system is - by design - modular.